lib/markdown-renderer.tsx

```
@@ -118,11 +130,16 @@ export function MarkdownRenderer({ content }: MarkdownRendererProps) {
-      return `<p class="text-gray-700 leading-relaxed mb-4">${trimmed}</p>`
+      return `<p class="text-gray-700 dark:text-gray-200 leading-relaxed mb-4">${trimmed}</p>`
     })
 
     return finalProcessedLines.join("\n")
   }
 
-  return <div className="prose prose-lg max-w-none" dangerouslySetInnerHTML={{ __html: renderMarkdown(content) }} />
+  return (
+    <div
+      className="prose prose-lg max-w-none dark:prose-invert"
+      dangerouslySetInnerHTML={{ __html: renderMarkdown(content) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)